### ISSUE TITLE
OS X desktop switch when running Nightmare

### ISSUE THREAD

**Issue by MiguelOller** [segmentio/nightmare#434]
Say that there's a Nightmare script called `index.js`. When running  `node index.js` from the terminal (OS X) in full screen and passing `{ show: true }` to the Nightmare constructor, it will switch to the first virtual desktop and open the Electron window. This works as expected.

If you pass nothing or `{ show: false }` it will still switch to the first virtual desktop even though it doesn't open an Electron window. This is very annoying because you have to jump back to the terminal every time you test the script. Is this an easy fix?

**Comment by emars** [comment]
I'm using iTerm2 in fullscreen mode and couldn't reproduce this.

Also tried it with the regular terminal.

Here is my code:
```js
var Nightmare = require('nightmare');
var vo = require('vo');

vo(function* (){
  var nightmare = new Nightmare({ });

  yield nightmare
    .goto('http://reddit.com')


  yield nightmare.end();
  return;
})(function(err, res){ });
```

**Comment by MiguelOller** [comment]
I ran your code with the regular terminal in my machine (OS X El Capitan Version 10.11.2) and it still happened.

Can anybody else reproduce this on their machine?

**Comment by MiguelOller** [comment]
Fixed with #452!
